tests/helpers/volumes.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+
+# Copyright (c) IBM Corporation 2023
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+import pytest
+
+class Volume:
+# Class volume to save the information with volume name and a flag of use
+# to mange in all test to not harcoded the values of the array.
+    def __init__(self, name):
+        self.name = name
+        self.in_use = False
+
+    def use(self):
+        self.in_use = True
+
+    def free(self):
+        self.in_use = False
+
```

Comment:
Question: should you add a __repr__ or a __str__ function, so that the content and volume of the class can be shown?  Not a requirement, but a suggestion.

Reply:
This might be useful for test logging which we will definitely need to improve since now both data set names and volume assignation will be dynamic for the test suite.

Reply:
I can see a __str__ function to dump the object , thus resolved.
```
   def __str__(self):
        return f'The volume {self.name} is in {self.in_use} in use'
```